Cython/Utility/CommonStructures.c

```diff
@@ -7,98 +7,90 @@ static PyObject* __Pyx_FetchCommonTypeFromSpec(PyType_Spec *spec, PyObject *base
 
 /////////////// FetchCommonType ///////////////
 
-static PyTypeObject* __Pyx_FetchCommonType(PyTypeObject* type) {
-    PyObject* fake_module;
-    PyTypeObject* cached_type = NULL;
-
-    fake_module = PyImport_AddModule((char*) "_cython_" CYTHON_ABI);
+static PyObject *fetch_fake_module(void) {
```

Comment:
Or, rather: `__Pyx_FetchSharedCythonABIModule()`. It's not fake at all, it even serves a purpose. :)

Reply:
Done